yolov3/yolov3/ViewControllers/DroneViewController.swift

```diff
@@ -121,39 +136,38 @@ class DroneViewController: UIViewController, CLLocationManagerDelegate {
       alert.addAction(UIAlertAction(title: "Ok", style: .default, handler: nil))
       self.present(alert, animated: true)
   }
-  
+   
   override var preferredStatusBarStyle: UIStatusBarStyle {
     return .lightContent
   }
-}
-
-
-public class SSID {
-    class func fetchNetworkInfo() -> [NetworkInfo]? {
-        if let interfaces: NSArray = CNCopySupportedInterfaces() {
-            var networkInfos = [NetworkInfo]()
-            for interface in interfaces {
-                let interfaceName = interface as! String
-                var networkInfo = NetworkInfo(interface: interfaceName,
-                                              success: false,
-                                              ssid: nil,
-                                              bssid: nil)
-                if let dict = CNCopyCurrentNetworkInfo(interfaceName as CFString) as NSDictionary? {
-                    networkInfo.success = true
-                    networkInfo.ssid = dict[kCNNetworkInfoKeySSID as String] as? String
-                    networkInfo.bssid = dict[kCNNetworkInfoKeyBSSID as String] as? String
-                }
-                networkInfos.append(networkInfo)
-            }
-            return networkInfos
+  
+  internal func receivedDisplayableFrame(_ frame: CVPixelBuffer) {
+      var cgImage: CGImage?
+      VTCreateCGImageFromCVPixelBuffer(frame, options: nil, imageOut: &cgImage)
+      
+      if let cgImage = cgImage {
+          DispatchQueue.main.async {
+              self.videoView.image = UIImage(cgImage: cgImage)
+          }
+      } else {
+          print("Video stream fail")
+      }
+  }
+  
+  func startStreamServer() {
+      DispatchQueue.global(qos: .userInteractive).async {
+        var currentImg: [UInt8] = []
+        while self.checkConnection() {
```

Comment:
i thought about that, but:
what if connection will be interrupted? self.isConnected will be still true...
and we need alert that connection is lost, which checkConnection() does.
we may refactor logic more deeply to obtain  self.isConnected
because currently it wouldnt work.